gemhardware/managers/include/gem/hw/managers/amc/AMCManager.h

```diff
@@ -1,41 +1,40 @@
-/** @file GLIBManager.h */
+/** @file AMCManager.h */
 
-#ifndef GEM_HW_GLIB_GLIBMANAGER_H
-#define GEM_HW_GLIB_GLIBMANAGER_H
+#ifndef GEM_HW_AMC_AMCMANAGER_H
+#define GEM_HW_AMC_AMCMANAGER_H
 
 #include <array>
 
 #include "gem/base/GEMFSMApplication.h"
-// #include "gem/hw/glib/GLIBSettings.h"
-
-#include "gem/hw/glib/exception/Exception.h"
+// #include "gem/hw/amc/AMCSettings.h"
 
 #include "gem/utils/soap/GEMSOAPToolBox.h"
-#include "gem/utils/exception/Exception.h"
 
 namespace gem {
   namespace hw {
     namespace glib {
-
       class HwGLIB;
-      class GLIBManagerWeb;
-      class GLIBMonitor;
+    }
+    namespace amc {
+
+      class AMCManagerWeb;
+      class AMCMonitor;
 
-      using glib_shared_ptr = std::shared_ptr<HwGLIB>;
+      using amc_shared_ptr = std::shared_ptr<gem::hw::glib::HwGLIB>;
```

Comment:
what/how many errors exist if this is just changed to the `amc::HwGenericAMC`?
if it's few, i'd make that change now, rather than later

Reply:
Switching to `amcL::HwGenericAMC` was among the first things I tried, but it gave a handful of errors. It is mostly missed functionality. I'm not sure whether this functionality is still *required for operations*, but given the time constraints I went the easiest way. I believe it can be refactored rather quickly later. Also I'm not 100% sure, but I think there could be still some pure virtual methods in `amc::HwGenericAMC` which still require implementation migration from `HwGLIB`.